uaclient/api/u/pro/security/fix/_common/plan/v1.py

```diff
@@ -681,6 +687,7 @@ def _fix_plan_cve(issue_id: str, cfg: UAConfig) -> FixPlanResult:
 
     if livepatch_cve_status:
         fix_plan = get_fix_plan(title=issue_id)
+        fix_plan.current_status = str(FixStatus.SYSTEM_NON_VULNERABLE)
```

Comment:
using `str` adds an extra layer of indirection - I'd rather do `FixStatus.SYSTEM_NON_VULNERABLE.msg`